builtin/types/dposv3/dposv3.proto

```diff
@@ -405,8 +401,9 @@ message DposReferrerRegistersEvent {
 
 message DposDelegatorClaimsRewardsEvent {
     Address delegator = 1;
-    Address validator = 2;
-    BigUInt amount = 3;
-    int64 time = 4;
-    int64 height = 5;
+    repeated Address validators = 2;
+    repeated BigUInt amounts = 3;
+    BigUInt total_rewards_claimed = 4;
+    int64 time = 5;
+    int64 height = 6;
```

Comment:
We don't need `time` and `height` as `EventData` that wraps this event already has these fields. https://github.com/loomnetwork/go-loom/blob/60edf7d5ec947392a9e923dbe0fea9a2412770d6/plugin/types/types.proto#L76